### PR TITLE
Modified generated gSoap default enum values from 0 to -1.

### PR DESCRIPTION
**Pull request by ASUMike**

### CODE REVIEW

File: rest-server/src/gSoapFiles/CAP/soapH.h [hunk 1]
```diff
@@ -840,7 +840,8 @@ inline void soap_default__ns2__alert_status(struct soap *soap, enum _ns2__alert_
 #ifdef SOAP_DEFAULT__ns2__alert_status
 	*a = SOAP_DEFAULT__ns2__alert_status;
 #else
-	*a = (enum _ns2__alert_status)0;
+//	*a = (enum _ns2__alert_status)0;
```
Comment (kchoadley) [inline rest-server/src/gSoapFiles/CAP/soapH.h]: Remove this holdover line

File: rest-server/src/gSoapFiles/CAP/soapH.h [hunk 2]
```diff
@@ -922,9 +923,9 @@ inline _ns2__alert_info_area_geocode * soap_new__ns2__alert_info_area_geocode(st
 }
 
 inline _ns2__alert_info_area_geocode * soap_new_req__ns2__alert_info_area_geocode(
-	struct soap *soap,
-	const std::string& valueName,
-	const std::string& value)
+		struct soap *soap,
```
Comment (kchoadley) [inline rest-server/src/gSoapFiles/CAP/soapH.h]: There's a lot of non-functional changes from here down. Just indentations from what I can tell. Is there a reason for them?

File: rest-server/src/gSoapFiles/CAP/soapH.h (hunk 1 above)
Comment (ASUMike) [reply rest-server/src/gSoapFiles/CAP/soapH.h]: done

File: rest-server/src/gSoapFiles/CAP/soapH.h (hunk 2 above)
Comment (ASUMike) [reply rest-server/src/gSoapFiles/CAP/soapH.h]: done

**Pull request merged**